widget/check.go

```diff
@@ -284,6 +284,12 @@ func (c *Check) TypedRune(r rune) {
 // TypedKey receives key input events when the Check is focused.
 func (c *Check) TypedKey(key *fyne.KeyEvent) {}
 
+// SetText sets the text of the Check
```

Comment:
Thanks for this, you would add since in the comment before the function definition, using:
```go
//
// Since: v2.4
```